### PR TITLE
Feature/venue testcases

### PR DESCRIPTION
**Pull request by Backiaraj07**
Completed Test case for Venues

And Created Data model for Refund process.

### CODE REVIEW

File: BookingSystem.Tests/BAL.Tests/VenueServiceTest/DeleteVenueTest.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BookingSystem.UnitTests.BAL.Tests.VenueServiceTest
+{
+    public class DeleteVenueTest
+    {
+
+    }
```
Comment (bhairab15) [inline BookingSystem.Tests/BAL.Tests/VenueServiceTest/DeleteVenueTest.cs:12]: Why this class remained empty ?

File: BookingSystem.Tests/BAL.Tests/VenueServiceTest/GetVenueTest.cs [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace BookingSystem.UnitTests.BAL.Tests.VenueServiceTest
+{
+    public class GetVenueTest
+    {
+        private Mock<IVenueRepository> _venuerepository;
+
```
Comment (bhairab15) [inline BookingSystem.Tests/BAL.Tests/VenueServiceTest/GetVenueTest.cs:24]: Naming convention should remain consistent. Some places it is _mock

File: BookingSystem.Tests/Integration.Tests/VenueRepositoryTest/UpdateVenueTest.cs [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+                .AddJsonFile("appunittestsettings.json")
+                .Build();
+            return config;
+        }
+        [Fact]
+
+        public async Task UpdateVenue_ReturnOKResult_WithVenueId()
+        {
+
+
```
Comment (bhairab15) [inline BookingSystem.Tests/Integration.Tests/VenueRepositoryTest/UpdateVenueTest.cs:52]: Empty method?

File: BookingSystemAPI/Controllers/VenueController.cs [hunk 4]
```diff
@@ -38,7 +38,7 @@ public async Task<ActionResult> Get()
             try
             {
```
Comment (bhairab15) [inline BookingSystemAPI/Controllers/VenueController.cs:39]: At the bottom still we have a hard coded string

File: BookingSystem.Tests/BAL.Tests/VenueServiceTest/GetVenueTest.cs (hunk 2 above)
Comment (Backiaraj07) [reply BookingSystem.Tests/BAL.Tests/VenueServiceTest/GetVenueTest.cs:24]: Changed name convention.

File: BookingSystem.Tests/Integration.Tests/VenueRepositoryTest/UpdateVenueTest.cs (hunk 3 above)
Comment (Backiaraj07) [reply BookingSystem.Tests/Integration.Tests/VenueRepositoryTest/UpdateVenueTest.cs:52]: written the update testcase

File: BookingSystem.Tests/BAL.Tests/VenueServiceTest/DeleteVenueTest.cs (hunk 1 above)
Comment (Backiaraj07) [reply BookingSystem.Tests/BAL.Tests/VenueServiceTest/DeleteVenueTest.cs:12]: Yet to write.

File: BookingSystemAPI/Controllers/VenueController.cs (hunk 4 above)
Comment (Backiaraj07) [reply BookingSystemAPI/Controllers/VenueController.cs:39]: Declared in common place, and implemented

**Review by bhairab15 — APPROVED** [review]

**Pull request merged**